.ci.yaml

```diff
@@ -141,6 +141,25 @@ platform_properties:
         {
           "sdk_version": "15a240d"
         }
+  mac_android_benchmark:
+    properties:
+      contexts: >-
+        [
+          "osx_sdk"
+        ]
+      dependencies: >-
+        [
+          {"dependency": "apple_signing", "version": "version:to_2024"}
+        ]
+      device_type: "msm8952"
+      mac_model: "Macmini8,1"
```

Comment:
https://chromium-swarm.appspot.com/botlist?c=id&c=task&c=cpu&c=device_type&c=mac_model&c=os&c=pool&c=status&d=asc&f=device_type%3Amsm8952&f=os%3AMac&k=os&s=id

`Mac_arm64_android` is running on arm, so to make it equivalent I think you want `"Macmini9,1"` here.

![Screenshot 2024-01-26 at 3 17 08 PM](https://github.com/flutter/flutter/assets/682784/61cb5e36-b5de-4fd6-9133-a6dd38993f67)

Reply:
What do you think about adding a description for what types of tests should use this host so that as people add new tests and are less familiar with the naming convention or constraints they have something in file to reference? 

My best attempt below.

```
Mac host with android sdk intended for test that care about HOST consistency but do not care about  TARGET architecture.  
```